src/sentry/static/sentry/app/views/eventsV2/results.tsx

```diff
@@ -68,14 +69,15 @@ class Results extends React.Component<Props, State> {
     return {...prevState, eventView};
   }
 
-  state = {
+  state: State = {
     eventView: EventView.fromLocation(this.props.location),
     error: '',
     errorCode: 200,
     totalValues: null,
     showTags: readShowTagsState(),
     needConfirmation: false,
     confirmedQuery: false,
+    incompatibleQuery: null,
```

Comment:
I think this should probably be more descriptive of being incompatible specifically to alert creation.

So maybe `incompatibleAlertNotice`